<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailUpdateWindow</class>
 <widget class="QDialog" name="DetailUpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_15">
   <item>
    <widget class="QGroupBox" name="groupBoxDetailed">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Üyelik Bilgileri</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0,0,0,0,5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_16">
                 <item>
                  <widget class="QLabel" name="labelMemberFoto">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Box</enum>
                   </property>
                   <property name="text">
                    <string>Foto</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1">
                   <item>
                    <widget class="QToolButton" name="toolButtonAddPhoto">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="layoutDirection">
                      <enum>Qt::LeftToRight</enum>
                     </property>
                     <property name="text">
                      <string>Fotoğraf</string>
                     </property>
                     <property name="icon">
                      <iconset resource="../Resources/bys.qrc">
                       <normaloff>:/Fotos/pic.ico</normaloff>:/Fotos/pic.ico</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                     <property name="toolButtonStyle">
                      <enum>Qt::ToolButtonTextBesideIcon</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditMemberId">
                 <property name="inputMethodHints">
                  <set>Qt::ImhDigitsOnly</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateTimeEdit" name="dateTimeEditMembershipDate">
                 <property name="date">
                  <date>
                   <year>2015</year>
                   <month>3</month>
                   <day>19</day>
                  </date>
                 </property>
                 <property name="displayFormat">
                  <string>dd.MM.yyyy</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxMemberType">
                 <item>
                  <property name="text">
                   <string>Asli</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Fahri</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxMemberStatus">
                 <item>
                  <property name="text">
                   <string>Aktif</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Pasif</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Kişisel Bilgileri</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Adı :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Soyadı :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_35">
                   <property name="text">
                    <string>Baba Adı :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>Doğum Yeri : </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>Doğum Tarihi : </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Cinsiyet :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_22">
                   <property name="text">
                    <string>Medeni Hali :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Cep Tel :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>E-Posta :</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLineEdit" name="lineEditName">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditSurname">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditNameFathername">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditNameBirthPlace">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="dateTimeEditBirthDate">
                   <property name="date">
                    <date>
                     <year>2015</year>
                     <month>3</month>
                     <day>19</day>
                    </date>
                   </property>
                   <property name="displayFormat">
                    <string>dd.MM.yyyy</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBoxSex">
                   <item>
                    <property name="text">
                     <string>Kadın</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Erkek</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBoxMaritalStatus">
                   <item>
                    <property name="text">
                     <string>Bekar</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Evli</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditCellPhone">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditMail">
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14" stretch="1,0">
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Eğitim Bilgileri</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Eğitim Durumu</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxEducationStatus">
                 <item>
                  <property name="text">
                   <string>Mezun</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Öğrenci</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Terk</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Derecesi</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxEducationLevel">
                 <item>
                  <property name="text">
                   <string>İlkokul</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Ortaokul</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Lise</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Önlisans</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Lisans</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Lisansüstü</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Doktora</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Son Mezun Olunan</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Okul :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>Bölüm :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Mesleği :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="lineEditSchoolName">
                     <property name="clearButtonEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditDepartmentName">
                     <property name="clearButtonEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditOccup">
                     <property name="clearButtonEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Kan Grubu</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QComboBox" name="comboBoxBlood">
                 <item>
                  <property name="text">
                   <string>A</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>B</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>AB</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxRhFactor">
                 <item>
                  <property name="text">
                   <string>Rh(+)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Rh(-)</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Adres Bilgileri</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>Ev Adresi : </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>98</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="textEditHomeAddress"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13" stretch=""/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="2,7">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Kurum Adı :</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditCompanyName">
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,7">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string>Kurum Adresi : </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="textEditWorkAddress"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="2,7">
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Kurum Tel :</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditWorkPhone">
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17" stretch="5,0">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonUpdate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Güncelle</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/bys.qrc">
              <normaloff>:/Fotos/refresh.ico</normaloff>:/Fotos/refresh.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/bys.qrc"/>
 </resources>
 <connections/>
</ui>
